BEIMA.Client/src/shared/ItemList/ItemList.js

```diff
@@ -48,7 +48,7 @@ const Item = ({item, RenderItem}) => {
   return (
     <div className={styles.item}>
       <div className={styles.row}> 
-        <div className={styles.itemName}>{item.deviceTag} - {"<Device Type Name>"} - {"<Building Name>"}</div>
+        <div className={styles.itemName}>{item.deviceTag} - {item.name} - {item.buildingName}</div>
```

Comment:
I forgot the device type list page also uses this, I don't think building info is needed for a device type. I'm thinking we could pass in another argument that indicates if it's a device or device type, then we can return a div with building info and then one without. We can use the ternary syntax, maybe something like `{device ? <div with building info> : <div without building info>}`